### PR TITLE
Start preparation phase to convert audio domains gradle from Groovy  DSL to Kotlin DSL

### PR DESCRIPTION
**Pull request by oas004**
#### WHAT
Starting the preparation phase for converting the gradle files in the audio domain from Groovy DSL to Kotlin DSL.

#### WHY

See #833 

#### HOW

My plan here is to start with each domain and go through the preparation phases from: https://docs.gradle.org/current/userguide/migrating_from_groovy_to_kotlin_dsl.html#prepare_your_groovy_scripts

On this step, I will separate the gradle files in each domain by commits, so that if you want to review it gradle file by gradle file, you can do so by reviewing commit by commit.

(By domain here I mean audio is one domain, auth is one domain etc.)

Then step 2 is to actually convert each gradle module to Kotlin DSL and rename the files to start using it. For this step I will make a PR for each module.

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

**Comment by oas004** [comment]
If you guys feel that this approach will lead to to many small PRs I can of course change it :)

File: audio-ui/build.gradle [hunk 1]
```diff
@@ -15,79 +15,78 @@
...
+            res.srcDirs += "src/sharedTest/res"
         }
     }
     lint {
-        checkReleaseBuilds false
-        textReport true
-        disable 'MissingTranslation', 'ExtraTranslation'
+        checkReleaseBuilds = false
+        textReport = true
+        disable "MissingTranslation", "ExtraTranslation"
```
Comment (luizgrp) [inline audio-ui/build.gradle:83]: I wonder if this is a function invocation?

**Review by luizgrp — APPROVED** [review]
LGTM

File: audio-ui/build.gradle (hunk 1 above)
Comment (oas004) [reply audio-ui/build.gradle:83]: Yeah I think you are right! Can I change it when I convert the file to .kts? :)

Comment (luizgrp) [reply audio-ui/build.gradle:83]: sure!

**Pull request merged**